src/shared/lib/datetime/iso/cast.js

```diff
@@ -0,0 +1,13 @@
+import { ZoneId, ZonedDateTime } from '@js-joda/core';
+import '@js-joda/timezone/dist/js-joda-timezone-10-year-range';
+
+/**
+ * Returns locale-cast YYYY-MM-DD from a UTC-cast 8601Z
+ * @param {*} tz IANA timezone string, e.g. `America/Los_Angeles`
+ * @param {*} utc ISO8601Z string, e.g. `2020-04-02T01:23:45:678Z`
```

Comment:
This is a super minor thing, but what is `:678Z` at the end? 

In my understanding a standard ISO datetime is: `2020-04-02T12:18:17+02:00`

There could be fractional seconds, which I think we shouldn't be using, so `2020-04-02T12:18:17.912436+02:00` is also valid but probably wrong for our usecase.

Reply:
BTW, this is also cutting it to data if we have `.split('T')[0]`, so it's actually returning something different.

Reply:
8601 is not just a single format, it covers a wide variety of implementations; `2020-04-02T12:18:17+02:00` is one such implementation. I typoed a colon instead of a period there, but see: `new Date().toISOString() // '2020-04-02T13:47:23.243Z'`

Reply:
OK, clear now.